libraries/rush-lib/package.json

```diff
@@ -57,7 +57,8 @@
     "tapable": "2.2.1",
     "tar": "~6.1.11",
     "true-case-path": "~2.2.1",
-    "uuid": "~8.3.2"
+    "uuid": "~8.3.2",
+    "pnpm-sync": "~0.0.3"
```

Comment:
The NPM system has a design flaw where, if an NPM package exposes a CLI bin script `xyz`, that bin script must in all circumstances get created as `node_modules/.bin/xyz`.  Therefore, if two different versions need to be installed side-by-side, then the package manager will complain and print warnings, because there is only one `xyz` file path.  (Which version should get installed there? What if you wanted to invoke the other version?) To avoid this problem, CLI packages should never get installed as indirect dependencies, because that is how side-by-side versions arise.

As a result, the general practice is to make separate NPM packages when you have both an API library (`@microsoft/rush-lib`, `@angular/core`, `webpack`) and a CLI command (`@microsoft/rush`, `@angular/cli`, `webpack-cli`).  

Maybe we should create a separate `pnpm-sync-lib` or `pnpm-sync-cli` package?